plugin/src/main/java/com/dff/cordova/plugin/camera/dagger/DaggerManager.java

```diff
@@ -28,38 +31,75 @@
  */
 public class DaggerManager {
     private static DaggerManager daggerManager;
+    
+    // contains dangerous permissions
+    // @see https://developer.android.com/guide/topics/permissions/overview.html#normal-dangerous
+    public static final String[] PERMISSIONS = new String[] {
+        Manifest.permission.CAMERA,
+    };
 
     private AppComponent appComponent;
     private ActionHandlerServiceComponent actionHandlerServiceComponent;
     private PluginComponent pluginComponent;
+    private ActivityComponent activityComponent;
     private CameraActivityComponent cameraActivityComponent;
```

Comment:
`cameraActivityComponent` und `previewActivityComponent` werden jedes Mal neu erzeugt und nicht mehr benötigt.

Reply:
Die Komponent werden in Zeile 105 bzw 115 benötigt, wenn man die jeweiligen Activitys injectet.

Reply:
Sehe ich nicht.